### DISCUSSION TITLE
enabling conda within neurodesktop

### DISCUSSION THREAD

**Discussion by fphsFischmeister** [neurodesk/neurodesk.github.io-hugo#377] [General]
Dear all,

We are currently implementing some custom Python scripts to analyze our fMRI data. While implementing, I noticed that conda is within the search path when using the JupyterLab environment but not when utilizing the Neurodesk platform. While I currently simply run /opt/conda/bin/conda init to initialize, I was wandering if there is any reason for not having conda  within the search path of Neurodesktop. Additionally, just to make sure, is the /opt/conda path the same instance within JupyterLab and Neurodesk?

Thanks in advance,

Florian

**Comment by stebo85** [comment]
Dear Florian, this sounds like bug. I investigate and get back to you. Which version of neurodesk are you using?


Thank you

Steffen

--

W: https://mri.sbollmann.net<https://mri.sbollmann.net/>  | W: https://www.neurodesk.org<https://www.neurodesk.org/>  | T: https://twitter.com/sbollmann_MRI  | G: https://github.com/stebo85

Book meeting: https://calendly.com/s-bollmann/meeting

________________________________
From: Florian Ph.S Fischmeister ***@***.***>
Sent: Monday, October 30, 2023 6:17:57 AM
To: NeuroDesk/neurodesk.github.io ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Neurodesk Organisation] enabling conda within neurodesktop (Discussion #377)


Dear all,

We are currently implementing some custom Python scripts to analyze our fMRI data. While implementing, I noticed that conda is within the search path when using the JupyterLab environment but not when utilizing the Neurodesk platform. While I currently simply run /opt/conda/bin/conda init to initialize, I was wandering if there is any reason for not having conda within the search path of Neurodesktop. Additionally, just to make sure, is the /opt/conda path the same instance within JupyterLab and Neurodesk?

Thanks in advance,

Florian

**Comment by fphsFischmeister** [comment]
We are using vnmd/neurodesktop:20230830.

Thanks,

Florian

**Reply by stebo85** [reply]
Dear @fphsFischmeister - I can confirm it's a bug. I just fixed it and we will create a new release.

**Comment by stebo85** [comment]
Dear @fphsFischmeister,

we created a new release. Can you try to the new release from here? https://www.neurodesk.org/docs/getting-started/neurodesktop/

when you use conda environments in neurodesktop make sure to follow these instructions to set it up: https://www.neurodesk.org/tutorials-examples/tutorials/programming/conda/

Let us know if this works as expected for you :)

**Reply by fphsFischmeister** [reply]
Dear @stebo85,
Tested the 2023-10-30 version and thanks everything works perfectly now.
Best, Florian

**Reply by stebo85** [reply]
That's great to hear! Thank you for finding the bug and reporting it :) Let us know if there is anything else that comes up!
